container/Exchange.js

```diff
@@ -32,6 +32,10 @@ export default () => {
   const [translation, setTranslation] = useState(undefined)
```

Comment:
I think it would be good to use null here as well.

Reply:
Donederydoo